### DISCUSSION TITLE
Are we recreating wikipedia/investopedia with (some of) the concepts wiki?

### DISCUSSION THREAD

**Discussion by olarubaj** [rufuspollock/awesome-crypto-critique#147] [Ideas]
> Is it a good idea to have definitions of basic concepts that can otherwise be found on wikipedia/investopedia (e.g. gold)?

## PROS

- having all the information in one place is convenient
- able to have short, nuanced definitions
- have a glossary source (technically) for hover-over etc

## CONS

- the concepts covered in the guide are widely known for quite a long period of time and so they have already been covered on many other sites and by many other people (i.e. on wikipedia/investopedia) so we're doing the same work yet again
- for sure it depends on the audience but I personally would much rather read on most of these concepts on wikipedia/investopedia and things that could actually draw my attention on the life itsef's site would be the articles that put the raw definitions in the context (e.g. political/economic); a curated list of resources (both internal and external) explaining the key concepts would also be nice

**Comment by rufuspollock** [comment]
@olarubaj good questions. My overall sense is that we the short summaries we have for well-known things like gold are distinct from wikipedia (and could reference wikipedia if needed) e.g. they are usually:

* very short
* may have a perspective relevant to the project here (e.g. if talking about gold or gold standard in general you would have different context that if defining for context of a crypto/web3 guide)
* having local content may permit things like "hover over" infoboxes etc

**Reply by theo-cox** [reply]
+1 to @rufuspollock. I'd further flag that our current internal capacity is devoted to more detailed explanations/FAQs, so we'll only be adding new concepts where these are really necessary

**Comment by SalBayat** [comment]
Wikipedia can be great as a primer or a general resource but their articles frequently have issues.  I recently had to remove a citation on central banking because someone had cited the Saylor Institute online class website for a fact relating to central banking in China.  Not only did the Saylor institute link not contain any information relevant regarding China and central banking, the linked article called into question the legitimacy of Central Banks.  

Curating articles and definitions on key topics that relate to crypto should be kept in house, as this will be more resistant to tampering and disinformation campaigns.
